README.md

```diff
@@ -23,8 +23,8 @@ and easy to use.
 
 ## For the Impatient
 
-I know, you just want to build and play.  If you have all the dependencies
-installed (see below), then simply do this:
+I know, you just want to build and play.  If you have all the [dependencies
+installed](#dependencies) (see below), then simply do this:
```

Comment:
In my opinion `installed` should not be part of the link.
If `dependencies` is a link then I find `(see below)` obsolete.
Another possibility would be to make `see below` a link.

Just my 2¢